Probesys/bileto
# new: Manage labels

## Related issue(s)



https://github.com/Probesys/bileto/issues/283

## How to test manually



1. As a super-admin (or admin with permission "Manage labels"), go to the admin > Labels
2. Create a label
3. Verify that you're redirected to the list of labels and that you see your label
4. Click on "Edit", change the name / description of the label, and submit
5. Verify that you're redirected to the list of labels and that the label has a the new name / description
6. Click on "Edit", and click on "Delete the label", confirm
7. Verify that you're redirected to the list of labels and that the label no longer exists

## Checklist
